portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,32 +27,39 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
+import java.util.List;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    ArrayList<String> jsonData = new ArrayList<String>();
+    
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    int numOfComments;
```

Comment:
Can you move this variable to inside the doGet method?

Reply:
Done.